### DISCUSSION TITLE
Can DocStrings be highlighted and coloured differently from other strings

### DISCUSSION THREAD

**Discussion by davesann** [clojure-vim/clojure.vim#30] [Q&A] [answered]
I find that the highlighting for doc-strings dominates what I can see and I want to grey these out. 

Other strings, part of the code,  I would leave unaltered. 

Is this possible? 

How would it be done?

**Comment by axvr** [comment, marked as answer]
We've actually had this question before: https://github.com/clojure-vim/clojure.vim/issues/28#issuecomment-1306852420

TL;DR: it's **_really_** difficult to do with the regular expressions Vim provides us.  In theory you could create additional tools which do that analyse the code and hook it up to Vim's "text properties" (see:`:help textprop.txt`).  Unfortunately such functionality is outside the scope of this project.

What might work as a partial-solution is to override the colour of all strings to something not as bright.  E.g.

```vim
" Highlight like another syntax class (like "character" in this example).
highlight link clojureString Character
highlight link clojureStringDelimiter Character

" Highlight with custom colour (red in this example).
highlight clojureString          guifg=#ff0000 ctermfg=1
highlight clojureStringDelimiter guifg=#ff0000 ctermfg=1
```

**Reply by davesann** [reply]
I thought that might be the case
